File: SGA_Api/SGA_Api/Models/Conteos/LecturaConteo.cs
namespace SGA_Api.Models.Conteos
{
    public class LecturaConteo
    {
        public Guid OrdenGuid { get; set; }
        public string CodigoAlmacen { get; set; } = string.Empty;
        public string? CodigoUbicacion { get; set; }
        public string? CodigoArticulo { get; set; }
        public string? DescripcionArticulo { get; set; }
        public string? LotePartida { get; set; }
        public decimal? CantidadContada { get; set; }
        public decimal? CantidadStock { get; set; }
        public string UsuarioCodigo { get; set; } = string.Empty;
        public DateTime Fecha { get; set; }
        public string? Comentario { get; set; }
        public Guid GuidID { get; set; } = Guid.NewGuid();
        public DateTime? FechaCaducidad { get; set; }

        // Campos para información de palet
        public Guid? PaletId { get; set; }
        public string? CodigoPalet { get; set; }
        public string? CodigoGS1 { get; set; }

        // Navigation property
        public OrdenConteo Orden { get; set; } = null!;
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/model/conteos/LecturaConteo.kt
package com.example.sga.data.model.conteos

data class LecturaConteo(
    val guidID: String,
    val ordenGuid: String,
    val codigoAlmacen: String,
    val codigoUbicacion: String,
    val codigoArticulo: String,
    val descripcionArticulo: String?,
    val lotePartida: String,
    val cantidadContada: Double?,
    val cantidadStock: Double?,
    val usuarioCodigo: String,
    val fecha: String,
    val comentario: String?,
    val fechaCaducidad: String?,
    val paletId: String? = null,
    val codigoPalet: String? = null,
    val codigoGS1: String? = null
)
